.gdbinit

```diff
@@ -1,4 +1,3 @@
 source .gdb_macros
 
-handle all nostop pass
```

Comment:
why this is removed?

Reply:
1. it's not needed
2. with it we ignore sigtrap in GDB by default which is terribly confusing